ironfish/src/memPool/recentFeeCache.ts

```diff
@@ -30,15 +30,18 @@ export class RecentFeeCache {
     this.chain = options.chain
   }
 
-  async setUpCache(): Promise<void> {
+  async setUp(): Promise<void> {
     // Mempool is empty
     let currentBlockHash = this.chain.latest.hash
 
     for (let i = 0; i < this.numOfRecentBlocks; i++) {
       const currentBlock = await this.chain.getBlock(currentBlockHash)
       Assert.isNotNull(currentBlock, 'No block found')
 
-      const lowestFeeTransactions = this.getLowestFeeTransactions(currentBlock)
+      const lowestFeeTransactions = this.getLowestFeeTransactions(
+        currentBlock,
+        this.numOfTxSamples,
+      )
       lowestFeeTransactions.forEach((tx) => {
```

Comment:
we can just use a for loop here too, we don't need to create a closure just for this.